ATL/schema.sql

```diff
@@ -0,0 +1,53 @@
+-- Passengers 
+ CREATE TABLE Passengers ( 
+	PassengerId INT PRIMARY KEY IDENTITY(1,1) NOT NULL, 
+    FirstName NVARCHAR(50) NOT NULL, 
+    LastName NVARCHAR(50) NOT NULL, 
+    Age INT NOT NULL 
+)
+ --Airlines 
+ CREATE TABLE Airlines(
+	AirlineId INT PRIMARY KEY IDENTITY(1,1) NOT NULL, 
+    AirlineName NVARCHAR(50) NOT NULL, 
+    Concourse CHAR(1) NOT NULL, CHECK (Concourse IN ('A', 'B', 'C', 'D', 'E', 'F', 'T'))
```

Comment:
With this structure we will have many different id's for the same Airline. How are we going to reference the Airline Delta from other tables if we can't uniquely identify it? Here you should have implemented a Many-to-Many relationship from Airline to a Concourse(different table).